gulp/tasks/scripts.js

```diff
@@ -202,17 +201,40 @@ function scriptsApps() {
   // Run each application's JS through webpack and store the gulp streams.
   const streams = [];
   apps.forEach( app => {
-    streams.push(
-      _processScript(
-        webpackConfig.appsConf,
-        `/apps/${ app }/js/**/*.js`,
-        `/apps/${ app }/js`
-      )
-    );
+
```

Comment:
Trivial: Remove blank line.

Reply:
Pretty sure this is autofixed to have the space in our current ESLint config for https://eslint.org/docs/rules/lines-around-comment, but I'll double-check shortly.

Reply:
If that's the case, perhaps we could set [allowBlockStart](https://eslint.org/docs/rules/lines-around-comment) to true?

Reply:
Yeah the linter adds space before comments. We'd need to adjust the linter settings if we wanted this to stick.

Reply:
Okay I'll test with `allowBlockStart`

Reply:
That works. Added here, and https://github.com/cfpb/cfgov-refresh/pull/3821 and https://github.com/cfpb/development/pull/161